### PR TITLE
feat: protx diff script

### PR DESCRIPTION
**Pull request by strophy**
## Issue being fixed or feature implemented

Core team requested a script specified as:
"Setup a script on 80% of nodes to run `protx diff` with random valid heights once per minute (via cron); continue doing this before and after fork is active; this is for cache invalidation; mimics mobile clients"

## What was done?

Implement a script according to this definition

## How Has This Been Tested?

On testnet

## Breaking Changes

None

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: ansible/roles/mn_protx_diff/files/protx-diff.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
+BLOCKCOUNT=$(dash-cli getblockcount)
+DIP3BLOCK=$(dash-cli getblockchaininfo | jq .bip9_softforks.dip0003.since)
+BASEBLOCK=$(shuf -i $DIP3BLOCK-$BLOCKCOUNT -n 1)
+ENDBLOCK=$(shuf -i $BASEBLOCK-$BLOCKCOUNT -n 1)
```
Comment (PastaPastaPasta) [inline ansible/roles/mn_protx_diff/files/protx-diff.sh:6]: Am I correct this is doing subtraction? I think the use of `protx diff` is using the actual heights such as `protx diff 100020 100000`

Comment (strophy) [reply ansible/roles/mn_protx_diff/files/protx-diff.sh:6]: This isn't subtraction, it's an argument to `shuf -i` which is `-i, --input-range=LO-HI   treat each number LO through HI as an input line`

Comment (PastaPastaPasta) [reply ansible/roles/mn_protx_diff/files/protx-diff.sh:6]: aah, okay :) sounds good

**Review by PastaPastaPasta — APPROVED** [review]
I think this looks correct

**Pull request merged**